config

```diff
@@ -0,0 +1,26 @@
+server {
+	listen 8081;
+	host 127.0.0.1;
+	server_name free_money_near_u;
```

Comment:
Is this part of a plan or just a random name? :-)

Reply:
just a random name, it does not matter at all